pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -545,6 +547,18 @@ func (rc *reconciler) updateDevicePath(volumesNeedUpdate map[v1.UniqueVolumeName
 	} else {
 		for _, attachedVolume := range node.Status.VolumesAttached {
 			if volume, exists := volumesNeedUpdate[attachedVolume.Name]; exists {
+				// If volume devicePath is already known from previous mount checking, no need to update it again.
+				if volume.devicePath != "" {
+					target, err := filepath.EvalSymlinks(attachedVolume.DevicePath)
+					if err != nil {
+						target = attachedVolume.DevicePath
+					}
+					if volume.devicePath != target {
+						glog.Warningf("The volume %q device path obtained from node status (%q) does not match with the actual devicePath (%q) from /proc/mounts",
```

Comment:
Let's use `kubeClient.CoreV1().Nodes().Get(...)` if it does not match mount.

Reply:
changed the code based on the suggestion.